sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/ExtractBenchmark.scala

```diff
@@ -74,8 +76,8 @@ object ExtractBenchmark extends SqlBasedBenchmark {
       field: String,
       from: String): Unit = {
     val expr = func match {
-      case "extract" => s"EXTRACT($field FROM ${castExpr(from)})"
-      case "date_part" => s"DATE_PART('$field', ${castExpr(from)})"
+      case "extract" => s"EXTRACT($field FROM ${castExpr(from)}) AS $field"
+      case "date_part" => s"DATE_PART('$field', ${castExpr(from)}) AS $field"
```

Comment:
Out of curiosity, why do we need to add alias?

Reply:
When I debugged this, I showed dataframes to terminal. And printed tables were so wide.